src/utils.js

```diff
@@ -10,17 +33,54 @@ const chunk = (array, chunkSize) => {
   return result;
 };
 
+const loadIncludesFromFile = () => {
+  console.log(`Used include file: ${includesFile}`);
+
+  const txtIncludes = includesFile;
+  const lines = readFileSync(txtIncludes, { encoding: 'utf8' }).split(/\r?\n/);
+
+  // sort by line length
+  // exclude comments
+  return lines
+    .sort((a, b) => b.length - a.length)
+    .filter(line => !line.startsWith('#'));
+};
+
+const loadExcludesFromFile = () => {
+  console.log(`Used exclude file: ${excludesFile}`);
+
+  const txtIncludes = excludesFile;
+  const lines = readFileSync(txtIncludes, { encoding: 'utf8' }).split(/\r?\n/);
+
+  // sort by line length
+  // exclude comments
+  return lines
+    .sort((a, b) => b.length - a.length)
+    .filter(line => !line.startsWith('#'));
+};
+
+const includes = loadIncludesFromFile()
+const excludes = loadExcludesFromFile()
+
 const readDir = (dir, filesList = []) => {
   const files = readdirSync(dir);
+
   for (let file of files) {
     const filePath = join(dir, file);
 
+    // in list of excluded files
+    const isExcluded = excludes.find(ex => filePath.includes(ex));
```

Comment:
```suggestion
    const isExcluded = excludes ? excludes.find(ex => filePath.includes(ex)) : false;
```

Reply:
I am for some reason unable to commit this suggestion via the github web ui but maybe you can? I believe this is a necessary companion to the other 2 you committed and it might actually be functional / CI would pass with just this addition